slack-strava/models/activity.rb

```diff
@@ -114,11 +119,17 @@ def total_elevation_gain_s
   end
 
   def pace_s
-    if type == 'Swim'
+    case type
+    when 'Swim'
       case team.units
       when 'km' then pace_per_100_meters_s
       when 'mi' then pace_per_100_yards_s
       end
+    when 'Ride', 'EBikeRide', 'VirtualRide'
+      case team.units
+      when 'km' then kilometer_per_hour_s
+      when 'mi' then pace_per_mile_s
```

Comment:
I realized now seeing #43 that this is inconsistent, showing speed for pace for metered units. Switched it to display both pace and speed in https://github.com/dblock/slack-strava/commit/d1b0fecd7f7990fc14bd98b99310d0c2b9a64693. I wonder whether we should choose which one depending on the type of activity or implement #27?